src/elements/content-sidebar/activity-feed/common/datestamp/Datestamp.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import * as React from 'react';
+import ReadableTime from '../../../../../components/time/ReadableTime';
+import { MILLISECONDS_PER_YEAR } from '../../../../../constants';
```

Comment:
Can we move this constant to this file, since it's only used here and in its tests?